Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/Course.java

```diff
@@ -34,6 +34,7 @@
 
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.tuple.Pair;
+import org.apache.logging.log4j.util.Strings;
```

Comment:
Why are you using the `isBlank` method from a Logging framework instead of the Apache Commons Language framework like in all the other places?